src/bottomsheet/bottomsheet.android.ts

```diff
@@ -117,6 +118,7 @@ function initializeBottomSheetDialogFragment() {
             if (options.options) {
                 const creationOptions = options.options;
                 this._transparent = creationOptions.transparent;
+                this._dismissOnDraggingDownSheet = creationOptions.dismissOnDraggingDownSheet;
```

Comment:
it would be better to do:
```ts
                this._dismissOnDraggingDownSheet = creationOptions.dismissOnDraggingDownSheet !== false;
```

Reply:
I just handled it like the parameter ```transparent```in the line before. Should this also be changed?

Reply:
I ll do it. Better to keep things separated